client/state/plugins/installed/selectors.js

```diff
@@ -210,6 +210,19 @@ export function isPluginActionInProgress( state, siteId, pluginId, action ) {
 	return isPluginActionStatus( state, siteId, pluginId, action, 'inProgress' );
 }
 
+/**
+ * Whether the plugin's status for one or more recent actions is completed.
+ *
+ * @param  {object}       state    Global state tree
+ * @param  {number}       siteId   ID of the site
+ * @param  {string}       pluginId ID of the plugin
+ * @param  {string|Array} action   Action, or array of actions of interest
+ * @returns {boolean}              True if one or more specified actions are completed, false otherwise.
+ */
+export function isPluginActionCompleted( state, siteId, pluginId, action ) {
```

Comment:
I'd say that if we only use this once, we might as well just use the `isPluginActionStatus` selector.

Reply:
I'm trying to be consistent with the code. There is not any call to `isPluginActionStatus()` out of the selectors context. `isPluginActionCompleted` was inspired on `isPluginActionInProgress`. I prefer keeping this selector, but I don't have s strong opinion about not removing it though. Please can you confirm removing it?

Reply:
I'm fine with keeping it if that's your preference, I have no strong feelings about removing it TBH 👍